src/deluge/model/mod_controllable/mod_controllable_audio.h

```diff
@@ -91,7 +91,7 @@ class ModControllableAudio : public ModControllable {
 	virtual bool setModFXType(ModFXType newType);
 	bool offerReceivedCCToLearnedParams(MIDIDevice* fromDevice, uint8_t channel, uint8_t ccNumber, uint8_t value,
 	                                    ModelStackWithTimelineCounter* modelStack, int32_t noteRowIndex = -1);
-	void receivedCCFromMidiFollow(ModelStack* modelStack, Clip* clip, int32_t ccNumber, int32_t value);
+	static void receivedCCFromMidiFollow(ModelStack* modelStack, Clip* clip, int32_t ccNumber, int32_t value);
```

Comment:
Since this function can be static it probably makes more sense to be part of midi follow than modcontrollable

Reply:
The original reason I had it in here was because that midi takeover calculation function is in here. I think I'll factor that out as well.